grammar/Makefile

```diff
@@ -13,3 +13,4 @@ generate_type_parser:
 clean:
 	rm -rf $(TYPE_OUTPUT_DIR)/*.py  $(TYPE_OUTPUT_DIR)/*.tokens $(TYPE_OUTPUT_DIR)/*.interp
 	rm -rf $(TESTCASE_OUTPUT_DIR)/*.py $(TESTCASE_OUTPUT_DIR)/*.tokens $(TESTCASE_OUTPUT_DIR)/*.interp
+	rm -rf ./*.tokens
```

Comment:
Also consider deleting the interp files which are typically used by interpreters.

Reply:
It is done in line 15. In the current folder it creates token file, not sure why it creates in two places.